extensions/ql-vscode/syntaxes/ql.tmLanguage.yml

```diff
@@ -668,6 +679,19 @@ repository:
     - match: '(?#simple-id)'
       name: entity.name.type.namespace.ql
 
+  # The `extends` clause of a class definition.
```

Comment:
```suggestion
  # The `implements` clause of a class definition.
```

Reply:
Doesn't `implements` only apply to modules?

Reply:
(I admit that would have been more obvious had I remembered to change the comment...)